spot_driver/launch/spot_driver.launch.py

```diff
@@ -163,10 +165,9 @@ def generate_launch_description() -> launch.LaunchDescription:
         )
     )
     launch_args.append(
-        DeclareLaunchArgument(
+        DeclareBooleanLaunchArgument(
             "launch_rviz",
             default_value="False",
```

Comment:
nit: to match the style of the other launch args
```suggestion
            default_value=False,
```